src/platforms/common/data-management/debug-files/source-context.mdx

```diff
@@ -58,6 +58,17 @@ For Flutter Android, iOS, and macOS, follow the instructions in the [Debug Symbo
 
 </PlatformSection>
 
+<PlatformSection supported={["react-native"]}>
+
+<Note>
+
+For React Native JavaScript code, follow the instructions in the [Source Maps](/platforms/react-native/sourcemaps/) guide.
+For native Android and iOS code, follow the instructions in the [Debug Symbols](/platforms/react-native/upload-debug/) guide.
+
+</Note>
```

Comment:
```suggestion

To get source context for React Native JavaScript code, follow the instructions in the [Source Maps](/platforms/react-native/sourcemaps/) guide.
For native Android and iOS code, follow the instructions in the [Debug Symbols](/platforms/react-native/upload-debug/) guide.
```

Reply:
I don't think a note makes this more digestible.

Ideally, I think this page should be tweaked to put all SDK-specific references inside `PlatformSection` tags, but that is out of scope for this PR.